packages/fxa-content-server/app/scripts/views/connect_another_device.js

```diff
@@ -26,8 +26,12 @@ import Template from 'templates/connect_another_device.mustache';
 import UserAgentMixin from '../lib/user-agent-mixin';
 import VerificationReasonMixin from './mixins/verification-reason-mixin';
 
-class ConnectAnotherDeviceView extends FormView {
-  template = Template;
+const ConnectAnotherDeviceView = FormView.extend({
```

Comment:
Oh I changed this to how Backbone classes are extended so that `events` below would work.